src/popover/types.js

```diff
@@ -174,6 +174,7 @@ export type PopoverPrivateStateT = {
   placement: TetherPlacementT,
   isLayerMounted: boolean,
   isMounted: boolean,
+  doAutoFocus: boolean,
```

Comment:
nit: to me, `doAutoFocus` does not really communicate the correct meaning. maybe consider `autoFocusAfterPositioning` or similar